list.php

```diff
@@ -30,16 +30,35 @@ function getFullName() {
 <div class="page-header">
     <h1><?php getFullName(); ?></h1>
 </div>
-
+<?php
+if($boxedlayout)
+{
+	$tablelayout = "col-md-3";
+}
+else
+{
+	$tablelayout = "col-md-6 col-lg-4";
```

Comment:
I don't think you should use `col-md-6` here because we add three columns. My bootstrap knowledge is pretty rusty, but I think more than 12 columns per row are not allowed. When disabling boxed layout, I can get into a sitation where things look a bit ugly. This might be connected.

![Screenshot from 2019-11-14 19-35-49](https://user-images.githubusercontent.com/16748619/68885920-5da52380-0716-11ea-902f-e87fe2f081bb.png)

Reply:
ARGH, you're totally right, should have been `col-md-4`